open_schools_platform/organization_management/circles/tests/utils.py

```diff
@@ -1,9 +1,19 @@
 from open_schools_platform.organization_management.circles.models import Circle
 from open_schools_platform.organization_management.circles.services import create_circle
+from open_schools_platform.organization_management.employees.tests.utils import create_test_employee
 from open_schools_platform.organization_management.organizations.services import create_organization
+from open_schools_platform.user_management.users.models import User
 
 
 def create_test_circle() -> Circle:
     organization = create_organization(name="test_org", inn="1111111111")
     circle = create_circle(organization=organization, name="test_circle", address="d", capacity=0, description="alalal")
     return circle
+
+
+def create_test_circle_with_user_in_org(user: User) -> Circle:
+    organization = create_organization(name="test_org", inn="1111111111")
+    employee = create_test_employee(user=user)
+    organization.employees.add(employee)
```

Comment:
should we save organization?

Reply:
No, we should not

Reply:
Or maybe we should, because as i can see, organization in this service has None value in employees field
I am going to test this once more.